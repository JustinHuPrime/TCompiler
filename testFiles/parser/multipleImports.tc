module foo;

import target;
import target::with::scope;