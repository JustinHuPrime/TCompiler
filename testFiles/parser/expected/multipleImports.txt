testFiles/parser/input/multipleImports.tc (code):
FILE(1, 1, STAB(), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), IMPORT(3, 1, ID(3, 8, target, REFERENCES())), IMPORT(4, 1, SCOPEDID(4, 8, target::with::scope, REFERENCES())))
